test/specs/repo/extensions/nuke.ts

```diff
@@ -1,49 +0,0 @@
-import LoginPage from 'wdio-mediawiki/LoginPage.js';
-import page from '../../../helpers/pages/page.js';
-
-describe( 'Nuke', function () {
-	beforeEach( async function () {
-		await browser.skipIfExtensionNotPresent( this, 'Nuke' );
-		await browser.waitForJobs();
-	} );
-
-	it( 'Should be able to delete a page through Special:Nuke', async function () {
-		await browser.editPage(
-			testEnv.vars.WIKIBASE_URL,
-			'Vandalism',
-			'Vandals In Motion'
-		);
-
-		const pageExistsResult = await browser.makeRequest(
-			testEnv.vars.WIKIBASE_URL + '/wiki/Vandalism',
-			{ validateStatus: false },
-			{}
-		);
-
-		expect( pageExistsResult.status ).toEqual( 200 );
-
-		await LoginPage.login(
-			testEnv.vars.MW_ADMIN_NAME,
-			testEnv.vars.MW_ADMIN_PASS
-		);
-		await page.open( '/wiki/Special:Nuke' );
-
-		await $( 'button.oo-ui-inputWidget-input' ).click();
-
-		await $( 'form li' );
-
-		await $( '.mw-checkbox-none' ).click();
-		await $( 'input[value="Vandalism"]' ).click();
-		await $( 'input[type="submit"]' ).click();
-
-		await browser.acceptAlert();
-
-		await expect( $( 'li*=has been queued for deletion' ) );
-	} );
-
-	it( 'Should show that a Nuke\'d page doesn\'t exist', async function () {
```

Comment:
I think this also tests JobRunner processing, which is part of the setup we create. So this might be still worth testing. But curious what other people think about this.

Reply:
same as above (you may have already changed it, sorry for the spam in that case): is the JobRunner processing being tested somewhere else or only here? 
1) if it's already being tested somewhere else I would vote to kick it this part too
2) otherwise, I would keep it but changing the name making it clear to any eye what is being tested